### PR TITLE
[no merge!] Re-enable 50_60 for Devnet

### PR DESCRIPTION
**Pull request by ogabrielides**
***Please remove the italicized help prompts before submitting or merging***

_Provide a general summary of your changes in the Title above_

_Pull requests without a rationale and clear improvement may be closed
immediately._

_Please provide clear motivation for your patch and explain how it improves
Dash Core user experience or Dash Core developer experience
significantly:_

* _Any test improvements or new tests that improve coverage are always welcome._
* _All other changes should have accompanying unit tests (see `src/test/`) or
  functional tests (see `test/`). Contributors should note which tests cover
  modified code. If no tests exist for a region of modified code, new tests
  should accompany the change._
* _Bug fixes are most welcome when they come with steps to reproduce or an
  explanation of the potential issue as well as reasoning for the way the bug
  was fixed._
* _Features are welcome, but might be rejected due to design or scope issues.
  If a feature is based on a lot of dependencies, contributors should first
  consider building the system outside of Dash Core, if possible._

## Issue being fixed or feature implemented
 - _Why is this change required? What problem does it solve?_
 - _If it fixes an open issue, please link to the issue here._

## What was done?
  _Describe your changes in detail_

## How Has This Been Tested?
  _Please describe in detail how you tested your changes._

  _Include details of your testing environment, and the tests you ran
to see how your change affects other areas of the code, etc._

## Breaking Changes
  _Please describe any breaking changes your code introduces_

## Checklist:
  _Go over all the following points, and put an `x` in all the boxes that apply._
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [ ] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Review by QuantumExplorer — APPROVED** [review]

File: src/llmq/options.cpp [hunk 1]
```diff
@@ -132,7 +132,7 @@ bool IsQuorumTypeEnabledInternal(Consensus::LLMQType llmqType, gsl::not_null<con
             return true;
         case Consensus::LLMQType::LLMQ_50_60:
             return !fDIP0024IsActive || !fHaveDIP0024Quorums ||
-                    Params().NetworkIDString() == CBaseChainParams::TESTNET;
+                    Params().NetworkIDString() == CBaseChainParams::TESTNET || Params().NetworkIDString() == CBaseChainParams::DEVNET;
```
Comment (knst) [inline src/llmq/options.cpp:135]: is it supposed to be merged to develop or you just need a build?

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has began to build this PR tagged as v21.0.0-devpr6139.0ef641a4. A new comment will be made when the image is pushed.

**Comment by knst** [comment]
I run build, the docker image would be available in ~1 hour

btw:
quorum `llmq_60_75` doesn't fit because `useRotation = true`
quorum `devnet_dip0024` doesn't fit too because `useRotation = true`

Seems as `llmq_50_60` is the only choice, because it is not rotating and has proper size.

**Review by PastaPastaPasta — CHANGES_REQUESTED** [review]
dnm

**Pull request closed without merging**